versioned_hdf5/slicetools.py

```diff
@@ -25,12 +25,19 @@ def split_slice(s, chunk):
         new_step = step
         yield i, Slice(new_start, new_stop, new_step)
 
-def split_chunks(shape, chunk_size):
-    if len(shape) > 1:
-        raise NotImplementedError
+def split_chunks(shape, chunks):
```

Comment:
Please add a docstring for public functions

Reply:
Only the stuff in api.py should be considered public. Also the stuff in slicetools.py will be replaced with ndindex.

Reply:
> Also the stuff in slicetools.py will be replaced with ndindex.

This is fair.